compiler/rustc_const_eval/src/transform/check_consts/check.rs

```diff
@@ -146,6 +164,7 @@ impl<'mir, 'tcx> Qualifs<'mir, 'tcx> {
 
         let return_loc = ccx.body.terminator_loc(return_block);
 
+        self.rpo = Some(traversal::reverse_postorder(&ccx.body));
```

Comment:
One of regressions is in many-assoc-items benchmarks which contains large amount of trivial const items with a type that looks like it cannot have any of qualification here, so the checks probably stop at type level before reaching data flow computations.